ehcache-impl/src/main/java/org/ehcache/impl/internal/loaderwriter/writebehind/operations/WriteOperation.java

```diff
@@ -51,7 +57,16 @@ public WriteOperation(K k, V v, long creationTime) {
 
   @Override
   public void performOperation(CacheLoaderWriter<K, V> cacheWriter) throws Exception {
-    cacheWriter.write(key, value);
+    Map<K, Exception> failures = new HashMap<>();
```

Comment:
Why are we doing this conversion? It makes no sense. The thing only writes on key... creating a BulkCacheWritingException is just churning heap for no reason.